### PR TITLE
Improve wording of Developer and User Guide 

### PR DESCRIPTION
**Pull request by C-likethis123**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -16,25 +16,41 @@ endif::[]
 
 By: `CS2103T-T13-1`      Since: `Sep 2019`      Licence: `MIT`
 
+== Introduction
+Welcome to ORGANice! This is an organ matching desktop application that is specifically designed with fast typing
+hospital administration staff in mind. This developer guide is meant for developers who would like to understand the
```
Comment (WilliamRyank) [inline docs/DeveloperGuide.adoc]: does `for a fast typer hospital administartive staff` sounds better?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -16,25 +16,41 @@ endif::[]
...
+design and structure of ORGANice.
```
Comment (WilliamRyank) [inline docs/DeveloperGuide.adoc]: `infrastructure` better?

Comment (WilliamRyank) [reply docs/DeveloperGuide.adoc]: i mean `architecture`

**Comment by WilliamRyank** [comment]
No stories for `Model` and `Storage`

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -33,7 +33,7 @@ to get started.
 
 == Quick Start
 
-Below is a list of things to do to use the application:
+Below is a list of things to do to use the ORGANice:
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: remove 'the'

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -154,7 +154,7 @@ a person and the criteria for valid inputs:
 
 === Viewing help : `help`
 There may be times where you do not understand how ORGANice works, or you need a simple reminder on how to use
-ORGANice. This command allows you to access the user guide to ORGANice.
+ORGANice. To access the user guide of ORGANice, do the following:
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: maybe can add in "**you can** do the following:"

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -73,6 +91,9 @@ The sections below give more details of each component.
 
 [[Design-Ui]]
 === UI component
+This section explains the design of the UI component. +
+Figure 4 below depicts the classes in the UI component and their relationships to each other.
+It will be referred to in our explanation of the UI component design.
```
Comment (Yen-Peng) [inline docs/DeveloperGuide.adoc]: i think can remove line 96.

line 95: "their **relationship with one another**."

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -90,7 +111,9 @@ The `UI` component,
 
 [[Design-Logic]]
 === Logic component
-
+This section explains the design of the Logic component. +
+Figure 5 below depicts the classes in the Logic component and their relationships to each other.
+It will be referred to in our explanation of the Logic component design.
```
Comment (Yen-Peng) [inline docs/DeveloperGuide.adoc]: same here , think line 116 is not needed.
line 115: "their relationship with one another."

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -64,7 +80,9 @@ image::LogicClassDiagram.png[]
 [discrete]
 ==== How the architecture components interact with each other
 
-The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+This section explains the interactions between the different architecture components through an example.
```
Comment (Yen-Peng) [inline docs/DeveloperGuide.adoc]: "the interactions **amongst** the different architecture components **using** an example."

File: docs/UserGuide.adoc (hunk 4 above)
Comment (C-likethis123) [reply docs/UserGuide.adoc]: Changed

File: docs/UserGuide.adoc (hunk 3 above)
Comment (C-likethis123) [reply docs/UserGuide.adoc]: Changed

File: docs/DeveloperGuide.adoc (hunk 6 above)
Comment (C-likethis123) [reply docs/DeveloperGuide.adoc]: Changed

File: docs/DeveloperGuide.adoc (hunk 5 above)
Comment (C-likethis123) [reply docs/DeveloperGuide.adoc]: Changed

File: docs/DeveloperGuide.adoc (hunk 7 above)
Comment (C-likethis123) [reply docs/DeveloperGuide.adoc]: Changed

**Pull request merged**